docs/en/connector-v2/source/LocalFile.md

```diff
@@ -254,6 +254,72 @@ Specifies Whether to process data using the tag attribute format.
 
 Filter pattern, which used for filtering files.
 
+The filtering format is similar to wildcard matching file names in Linux.
```

Comment:
We cannot tell users about the ambiguous conclusion. Please tell users directly that we use Java regular expressions.